### PR TITLE
Create a BeanELResolver test for Full and Standalone BeanSupport

### LINKED ISSUE #25601: Expression Language 6.0: Make the dependency on the `java.beans.` package (and hence the `java.desktop` module) optional
Make the dependency on the `java.beans.` package (and hence the `java.desktop` module) optional. If the `java.beans.` package is not available, an internal implementation is used to support the BeanELResolver that just provides getter/setter support as all other JavaBeans functionality requires classes from the `java.beans.*` package.

Spec issue: `https://github.com/jakartaee/expression-language/issues/47`

Discussion about TCK and removal of java.desktop dependency: https://www.eclipse.org/lists/el-dev/msg00209.html

### PR DESCRIPTION
**Pull request by pnicolucci**
- [x] I have considered the risk of behavior change or other zero migration impact (https://github.com/OpenLiberty/open-liberty/wiki/Behavior-Changes).
- [x] If this PR fixes an Issue, the description includes "Fixes #FILLMEIN" or "Resolves #FILLMEIN" (verify `release bug` label if applicable: https://github.com/OpenLiberty/open-liberty/wiki/Open-Liberty-Conventions).
- [x] If this PR resolves an external Known Issue (including APARS), the description includes "Fixes #FILLMEIN" or "Resolves #FILLMEIN".

fixes #25601

### CODE REVIEW

**Comment by pnicolucci** [comment]
#build

**Comment by pnicolucci** [comment]
#libby #build

File: dev/io.openliberty.org.apache.jasper.expressionLanguage.6.0_fat/test-applications/EL60BeanELResolverTest.war/src/io/openliberty/el60/fat/beanelresolver/beans/TestBeanBeanInfo.java [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+import java.beans.SimpleBeanInfo;
+
+/**
+ * A SimpleBeanInfo to define the read and write methods for TestBean.
+ */
+public class TestBeanBeanInfo extends SimpleBeanInfo {
+
+    @Override
+    public PropertyDescriptor[] getPropertyDescriptors() {
+        System.out.println("PAN: getPropertyDescriptors()");
```
Comment (volosied) [inline dev/io.openliberty.org.apache.jasper.expressionLanguage.6.0_fat/test-applications/EL60BeanELResolverTest.war/src/io/openliberty/el60/fat/beanelresolver/beans/TestBeanBeanInfo.java]: Looks left in by accident

File: dev/io.openliberty.org.apache.jasper.expressionLanguage.6.0_fat/test-applications/EL60BeanELResolverTest.war/src/io/openliberty/el60/fat/beanelresolver/beans/TestBeanBeanInfo.java [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+        PropertyDescriptor testPropertyDescriptor = null;
+        try {
+            testPropertyDescriptor = new PropertyDescriptor("test", TestBean.class, "returnTest", "writeTest");
```
Comment (volosied) [inline dev/io.openliberty.org.apache.jasper.expressionLanguage.6.0_fat/test-applications/EL60BeanELResolverTest.war/src/io/openliberty/el60/fat/beanelresolver/beans/TestBeanBeanInfo.java:26]: This part is a bit difficult to grasp without the javadoc. Could you add a comment: 

// PropertyDescriptor(String propertyName, Class<?> beanClass, String readMethodName, String writeMethodName)

File: dev/io.openliberty.org.apache.jasper.expressionLanguage.6.0_fat/test-applications/EL60BeanELResolverTest.war/src/io/openliberty/el60/fat/beanelresolver/beans/TestBeanBeanInfo.java (hunk 1 above)
Comment (pnicolucci) [reply dev/io.openliberty.org.apache.jasper.expressionLanguage.6.0_fat/test-applications/EL60BeanELResolverTest.war/src/io/openliberty/el60/fat/beanelresolver/beans/TestBeanBeanInfo.java]: Thanks for catching that!

**Comment by pnicolucci** [comment]
@volosied changes are: https://github.com/OpenLiberty/open-liberty/pull/29884/commits/e0dd57ebcfe7faecadaa59abe460d31d0882db06

**Comment by pnicolucci** [comment]
#libby #build

**Review by volosied — APPROVED** [review]

**Pull request merged**